cowprotocol/cowswap
# feat(twap): launch darkly feature flag

# Summary

Closes #2550 

Using a new launch darkly feature flag `advancedOrdersEnabled`

When off:
- Hide it from the header menu
- Hide it from the form links
- Redirect `advanced-orders` to `swap`

# To Test

1. With the flag off open the page
* `Advanced orders` should not be visible on the header menu
* `Advanced orders` should not be visible on the form links
2. Change the url to `/#/1/advanced-orders/`
* Should redirect to `/#/1/swap`
3. Turn the feature flag on
* `Advanced orders` should be visible on the header menu
* `Advanced orders` should be visible on the form links
4. Change the url to `/#/1/advanced-orders/`
* Should load the advanced orders page
